packages/rizin/build.sh

```diff
@@ -0,0 +1,46 @@
+TERMUX_PKG_HOMEPAGE=https://rizin.re
+TERMUX_PKG_DESCRIPTION="UNIX-like reverse engineering framework and command-line toolset"
+TERMUX_PKG_LICENSE="LGPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=v0.6.2
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_pre_configure() {
+	# Certain packages are not safe to build on device because their
+	# build.sh script deletes specific files in $TERMUX_PREFIX.
+	if $TERMUX_ON_DEVICE_BUILD; then
+		termux_error_exit "Package '$TERMUX_PKG_NAME' is not safe for on-device builds."
+	fi
+}
+
+# Get the Android architecture
+ARCH=$(getprop ro.product.cpu.abi)
```

Comment:
This should be handled automatically by the build system already.
(Provided we are compiling from source, see comment below)